pwndbg/disasm/arch.py

```diff
@@ -185,6 +185,13 @@ def enhance(instruction: PwndbgInstruction, emu: Emulator = None) -> None:
                     )
                 emu = jump_emu = None
 
+        enhancer: DisassemblyAssistant = DisassemblyAssistant.assistants.get(
+            pwndbg.gdblib.arch.current, generic_assistant
+        )
+
+        # Don't disable emulation yet, as we can use it to read the syscall register
+        enhancer._enhance_syscall(instruction, emu)
```

Comment:
It's weird codecov is reporting this line as not covered by tests. Does tests/gdb-tests/tests/test_context_commands.py not exercise this line of code?

(You can see the codecov message in the files tab: https://github.com/pwndbg/pwndbg/pull/2205/files)

Reply:
Very strange - the function is definitely invoked, because a couple tests check for the syscall name on the context view which this function provides.